### PR TITLE
Add SquadCast Source and Faros destination's converter

### PR DESCRIPTION
**Pull request by Roma-Kyrnis**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

> Fix [#160]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: sources/squadcast-source/test/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+          }),
+        } as any,
+        'incidentId-123'
+      );
+    });
+    const source = new sut.SquadcastSource(logger);
+    const streams = source.streams({});
+
+    const incidentsStream = streams[1];
+    const incidentsIter = incidentsStream.readRecords(SyncMode.FULL_REFRESH);
```
Comment (yevgenius) [inline sources/squadcast-source/test/index.test.ts:142]: Can we support incremental sync as well? the api has date filter https://apidocs.squadcast.com/#3d00d5c6-6b9b-410c-a11b-0da72c60d419

Comment (Roma-Kyrnis) [reply sources/squadcast-source/test/index.test.ts:134]: I've added incremental sync

**Comment by cjwooo** [comment]
@Roma-Kyrnis You have a test failure, otherwise LGTM

File: sources/squadcast-source/resources/spec.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "SquadCast Spec",
+    "type": "object",
+    "required": [
+      "token"
+    ],
+    "additionalProperties": false,
+    "properties": {
+      "token": {
```
Comment (yevgenius) [inline sources/squadcast-source/resources/spec.json:12]: add the link to description https://apidocs.squadcast.com/?version=latest#c624b7d7-4b14-4a0a-ae65-d85298e3f2a9

Comment (Roma-Kyrnis) [reply sources/squadcast-source/resources/spec.json:12]: This token granted from Squadcast user's page. For example  https://app.squadcast.com/user/USER_ID

**Comment by cjwooo** [comment]
@Roma-Kyrnis for this source and the GoogleCalendar source, please add a README.md and a bootstrap.md. I've added the markdown files for your other sources for you to use as templates. The README should be mostly copy paste with "replace <source name> with <new source name>. The bootstrap document should be short, and list the streams of the source and anything specific to the source system that a developer would need to know (e.g. local Jenkins server required to test Jenkins source).

**Review by yevgenius — APPROVED** [review]

**Pull request merged**